instrumentation/runtime-metrics/library/runtime-metrics-library.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: "$rootDir/gradle/instrumentation-library.gradle"
+
+dependencies {
+  testImplementation project(':testing-common')
+  testImplementation group: 'org.assertj', name: 'assertj-core', version: '1.7.1'
```

Comment:
Can we use newer versions of these?

Reply:
heh. absolutely. I copied this from somewhere...so there might want to be a more wide upgrade.

Reply:
done